pkg/operator/reconcile.go

```diff
@@ -55,7 +57,31 @@ func ReconcileConsole(cr *v1alpha1.Console) error {
 	if _, _, err := ApplyOAuth(cr, rt); err != nil {
 		return err
 	}
-
 	return nil
+}
+
+func DeleteAllResources(cr *v1alpha1.Console) error {
+	var errs []error
+
+	if err := DeleteService(cr); errors.IsNotFound(err) {
```

Comment:
You have this backwards, you want `!errors.IsNotFound`

but better approach would be to always append, and then `return errutil.FilterOut(errutil.NewAggregate(errs), errors.IsNotFound)`

Reply:
derp. resolved.